This Bash script was developed to analyze the bacterial composition of multiple samples, focusing on the most abundant families. It processes [output files](data/report_kraken_F1.txt) generated by Kraken2, where each file contains taxonomic assignments of sequencing reads at various hierarchical levels. The script specifically extracts lines corresponding to the taxonomic rank "family" (denoted by the letter "F") and retrieves, for each, its name and the associated number of reads.

The absolute abundance of a family in a given sample corresponds to the total number of reads assigned to that family. These raw counts are obtained using awk and then aggregated across all samples to identify the 50 most abundant families. The threshold of 50 is arbitrary; fewer families will be used in the downstream R pipeline to improve legend readability. This approach allows the analysis to focus on dominant families while grouping rarer ones into an "Other" category. Such a strategy helps to avoid visual clutter and enhances clarity in graphical representations.

The script then builds an absolute abundance matrix, where rows represent bacterial families (including "Other") and columns represent samples. Each cell contains the number of reads assigned to a specific family in a specific sample.

In a second step, the script produces a relative (or normalized) abundance matrix. For each cell, abundance is expressed as the proportion of total reads in the sample:

Normalized abundance= (Reads for a family)/(Total reads for a sample)
â€‹
 
This normalization enables the comparison of taxonomic profiles across samples with varying sequencing depths. It is essential for comparative analyses, such as stacked bar plot visualizations or similarity analyses using dendrograms. The final output consists of two TSV files: one containing the absolute abundances (taxa_abundance_absolute_top50_families.tsv) and the other the normalized abundances (taxa_abundance_normalized_top50_families.tsv), both focused on the 50 most dominant bacterial families.

In summary, this script automates the transition from raw Kraken2 output to a synthetic representation of dominant bacterial families, thereby facilitating ecological and statistical interpretation of microbial community composition across samples.
