##Assembling the “clean” genomes of Pinna nobilis and Pinna rudis species
-

It is essential to obtain a complete genome from a healthy Pinna nobilis individual, which can serve as a reference for comparison with genomes from individuals infected by the parasite Haplosporidium pinnae.
The reference genome is assembled from transcriptomes of P. nobilis and P. rudis, composed of highly accurate short reads obtained from the NCBI SRA database. These transcriptomes are presumed to be free from H. pinnae contamination.

To ensure this, the raw data are first “cleaned” using [Kraken2](./Kraken_2.md#Kraken2) and SeqKit, which detect, identify, and remove potential bacterial contaminations from a database containing most of the known marine bacteria (see the [script](./Script_classification) section for more details on the script) . The quality of the transcriptomic data is then assessed using [FastQC](./FastQC.md#FastQC). Based on these quality reports, Trimmomatic, an open-source software, is used to trim low-quality bases from the beginning and/or end of the reads in order to correct alignment errors. A second FastQC analysis is performed to confirm the improvement in read quality.

High-quality reads are then assembled using Trinity, a de novo assembler that reconstructs transcriptomes from fragmented RNA-Seq reads without the need for a reference genome. The completeness of the assembled transcriptome is assessed using BUSCO, which evaluates whether the assembly is complete, fragmented, or incomplete by searching for conserved, single-copy orthologous genes across a specific taxonomic group.

If the assembly quality is deemed satisfactory, it is processed through ORF Finder (Open Reading Frame Finder) to identify open reading frames (ORFs) within the assembled sequences, allowing the prediction of potential coding regions from randomly fragmented genomic sequences.
The contaminated transcriptomes used in this study originate from various Mediterranean locations (Greece, Italy, Spain, and France). Unlike the previously used healthy transcriptomes, these are composed of long reads. Therefore, the genome assembly pipeline differs from that used for short reads, although the objective remains the same.

The sequencing quality of the contaminated transcriptomes is first assessed using FastQC. If the quality is acceptable, bacterial sequences are removed using Kraken2, as described earlier. Similarly, data quality is re-evaluated with FastQC, and low-quality bases are trimmed from the reads using Trimmomatic.High-quality reads are assembled de novo using Flye, an assembler optimized for long-read sequencing data. To reduce redundancy caused by assembly artifacts, Purge_dups is applied to remove duplicated contigs. Remaining contigs are scaffolded using P_RNA_Scaffolder, which leverages paired-end RNA-seq alignments to connect expressed regions belonging to the same locus. Assembly quality is evaluated using BUSCO for completeness and QUAST for structural metrics such as N50 and L50, which reflect assembly contiguity and fragmentation.

Structural gene annotation is performed with BRAKER3, which integrates transcriptomic (RNA-seq) and protein homology evidence to train ab initio predictors. BRAKER3 produces GFF3 files for gene models and FASTA files for coding sequences. Predicted proteins are then aligned using DIAMOND against a reference database (UniProt) to infer putative functions via homology.
