# Analysis of the bacterial taxonomic composition in tissues of Pinna nobilis and Pinna rudis
----------------------------------------------------------------------------------------------

This section describes the analysis of bacterial communities associated with *Pinna nobilis* and *Pinna rudis* tissues based on non-targeted RNA-seq data.

In parallel with the transcriptomic analysis of Pinna nobilis, a study of the bacterial diversity associated with the mantle tissues of these organisms was conducted to determine whether the bacterial communities differ between healthy individuals and those infected by the parasite Haplosporidium pinnae. To this end, output files generated by Kraken2 (see [Kraken2](../genomics/Kraken_2.md) section), containing the bacterial taxa identified in the raw RNA-seq reads, used here as a proxy for bacterial presence in the tissues. Each output file lists the bacterial taxa detected and the number of reads assigned to each (example [here](data/report_kraken_F1.txt)).

In the [bash script](Scripts_bash/Script_binary_table.sh), a minimum threshold of 10 reads was established to reduce the likelihood of false positives due to taxonomic misassignments. This threshold is arbitrary, as there is currently no consensus regarding metagenomic identification in non-targeted transcriptomic datasets generated by Illumina sequencing. However, since RNA-seq datasets are not optimized for microbial detection, the presence and abundance of bacterial taxa should be interpreted with caution. This is especially true in the absence of complementary DNA-based metagenomic validation

Bacterial taxa with read counts equal to or above this threshold were selected and used to construct a presence/absence matrix. In this matrix, columns represent the different samples and rows correspond to the identified bacterial taxa. The presence of a taxon in a sample is marked with a “1” in the corresponding cell, while its absence is marked with a “0”. A detailed explanation of the Bash script is provided in this section: [Explanation-01](Explanation/Explanation_1.md)

The resulting binary matrix summarizes the presence or absence of bacterial taxa across samples. The objective is to compare bacterial diversity among samples in relation to biological, ecological, or geographic factors. Beta diversity analysis is particularly appropriate for this type of data, as it quantifies variation in species composition between samples or sites of interest (Legendre & De Cáceres, 2013).

To visualize the dispersion index of our samples, we use the RStudio environment to perform beta diversity analysis and generate visualizations based on a binary presence/absence matrix of bacterial taxa. This approach allows us to analyze bacterial taxonomic diversity between samples and to conduct statistical tests assessing β-diversity differences across multiple sample groups. Since our samples originate from various localities (France, Spain, Greece, and Italy), we were able to compare bacterial community compositions across these geographic groups.

The complete script used here is available [here](Script_R/Script_beta.R).Similar to the previous Bash script, the '[Explanation-02](Explanation/Explanation_2.md)' section provides a detailed explanation of the code.

The R script above provides an initial overview of sample dispersion. A more refined R script has been developed [here](Script_R/Script_PCoA_upgrade.R) to generate higher-quality PCoA plots, as shown below (fig.1).

![Capture d'écran 2025-05-23 151954](https://github.com/user-attachments/assets/e606008a-4b07-4548-addd-045de60a875e)

Figure 1 : β-diversity analysis (PCoA - Jaccard) with group ellipses 

To further explore the data, a SIMPER analysis (Similarity Percentage Analysis) was also performed. This ecological method is used to identify which elements contribute most to the dissimilarity between two or more groups of samples, typically in terms of species or taxonomic composition. It therefore allows for the identification of taxa that most influence the bacterial taxonomic dispersion among individuals.The R script is also available [here](Script_R/Script_SIMPER.R), along with its explanation in the [Explanation-03](Explanation/Explanation_3.md) section.

To conclude the bacterial analysis, we highlighted the abundance of the most dominant bacterial families in each sample using a stacked bar plot linked to a dendrogram. This type of visualization allows for the analysis of bacterial community composition while revealing similarities among samples (fig.2).

![Capture d'écran 2025-05-26 093338](https://github.com/user-attachments/assets/8454e99a-37f3-469b-8ed7-934201bff5e2)

Figure 2 : Stacked bar plot with hierarchical clustering (dendrogram) showing bacterial composition and abundance in Pinna spp samples.

To generate this type of plot, we first created a [Bash script](Scripts_bash/Script_abundance.sh) to calculate the abundance of each bacterial family and to retain the 20 most abundant ones, grouping the others into an "Other" category ([Explanation-04](Explanation/Explanation_4.md)). We then developed an [R script](Script_R/Script_barplot_&_dendrogram.R) ([Explanation-05](Explanation/Explanation_5.md)) to visualize the resulting data as a bar plot and a dendrogram.

We observed the presence of bacterial families potentially pathogenic to Pinna spp., such as Bacillaceae. Based on a literature review, we identified approximately twenty families known to be pathogenic to bivalves: Vibrionaceae, Nocardiaceae, Rhodobacteraceae, Flavobacteriaceae, Aeromonadaceae, Enterobacteriaceae, Clostridiaceae, Mycoplasmataceae, Rickettsiaceae, Francisellaceae, Arcobacteraceae, Bacillaceae, Staphylococcaceae, Photobacteriaceae, Planococcaceae, Pseudoalteromonadaceae, Shewanellaceae, and Mycobacteriaceae.
We therefore adapted the previous [R script](Script_R/Script_barplot_&_dendrogram_pathogens.R) to focus on these families (when present in Pinna spp. samples) in order to specifically visualize the abundance of potentially pathogenic families in fan mussel samples.


